cmake/Modules/FindODBC.cmake

```diff
@@ -0,0 +1,215 @@
+#
+# Consults the following vars (if set):
+#     ODBC_PROVIDER - one of UnixODBC, iODBC, or MDAC
```

Comment:
If this module is capable of discovering MDAC, iODBC, UnixODBC, why do we need separate `FindMDAC.cmake`, `FindiODBC.cmake` and `FindUnixODBC.cmake` then ? If this is an umbrella-module that works relies on above modules, please explicitly note that in comments.

Reply:
Done.